### PR TITLE
Movie API types to a separate package

### PR DESCRIPTION
**Pull request by mwielgus**
So that they can be imported in k8s without handlers etc.

### CODE REVIEW

**Comment by vishh** [comment]
@cadvisorJenkinsBot: add to whitelist

File: sinks/external_test.go [hunk 1]
```diff
@@ -22,6 +22,7 @@ import (
 
 	kube_api "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
 	fuzz "github.com/google/gofuzz"
+	_ "k8s.io/heapster/api/v1/types"
```
Comment (vishh) [inline sinks/external_test.go]: Why is this necessary?

File: integration/heapster_api_test.go [hunk 2]
```diff
@@ -29,7 +29,7 @@ import (
 	apiErrors "github.com/GoogleCloudPlatform/kubernetes/pkg/api/errors"
 	"github.com/golang/glog"
 	"github.com/stretchr/testify/require"
-	heapster_api "k8s.io/heapster/api/v1"
+	heapster_api_types "k8s.io/heapster/api/v1/types"
```
Comment (vishh) [inline integration/heapster_api_test.go]: nit: rename this to `api_v1`.

File: sinks/external_test.go (hunk 1 above)
Comment (mwielgus) [inline sinks/external_test.go]: Not sure but without this the test fails. Probably a fuzzer checks what packages are initialized or so.

File: integration/heapster_api_test.go (hunk 2 above)
Comment (mwielgus) [inline integration/heapster_api_test.go]: Done.

File: sinks/external_test.go (hunk 1 above)
Comment (vishh) [inline sinks/external_test.go]: Hmm. Thats wierd!

**Comment by vishh** [comment]
Thanks for the PR @mwielgus! LGTM.

**Comment by mwielgus** [comment]
#490

File: sinks/external_test.go (hunk 1 above)
Comment (afein) [inline sinks/external_test.go]: I have observed this test being fuzzy even at the current master. Not sure what the cause is
